### PR TITLE
enable deformable_detr for PDFE

### PR DESCRIPTION
**Pull request by xczhai**
### Details:
 - *enable sin*
 - *enable cos*
 - *fix index dtype in top_k_v2*
    PDPD top_k_v2 spec: https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/topk_cn.html
    OpenVINO top_k_v2 spec: https://docs.openvino.ai/latest/openvino_docs_ops_sort_TopK_3.html

### CODE REVIEW

File: src/frontends/paddle/tests/test_models/gen_scripts/generate_cos.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        outs = exe.run(
+            feed={'x': x},
+            fetch_list=[out])
+
+        saveModel(name, exe, feedkeys=['x'], fetchlist=[out], inputs=[x], outputs=[outs[0]], target_dir=sys.argv[1])
+
+    return outs[0]
+
+def main():
+    x = np.random.rand(8, 24, 32).astype(data_type)
```
Comment (meiyang-intel) [inline src/frontends/paddle/tests/test_models/gen_scripts/generate_cos.py:35]: It's better to expand the value range of input

**Review by meiyang-intel — APPROVED** [review]

File: src/frontends/paddle/tests/test_models/gen_scripts/generate_cos.py (hunk 1 above)
Comment (xczhai) [reply src/frontends/paddle/tests/test_models/gen_scripts/generate_cos.py]: > It's better to expand the value range of input

update it.

**Comment by yuxu42** [comment]
@ceciliapeng2011 Could you please take a review? Thanks!

File: src/frontends/paddle/src/op/top_k_v2.cpp [hunk 2]
```diff
@@ -27,7 +27,7 @@ NamedOutputs top_k_v2(const NodeContext& node) {
     std::string sort_type = sorted ? "value" : "none";
     std::string mode = largest ? "max" : "min";
 
-    auto node_topk = std::make_shared<default_opset::TopK>(x, k_expected_node, axis, mode, sort_type);
+    auto node_topk = std::make_shared<default_opset::TopK>(x, k_expected_node, axis, mode, sort_type, element::i64);
```
[the remark this answers is not in the document]
Comment (xczhai) [reply src/frontends/paddle/src/op/top_k_v2.cpp:30]: @ceciliapeng2011 
- In most cases, we will not use `index` so we don't run into this issue.
- The spec of topk shows the default dtype of index is i64. https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/topk_cn.html#fanhui

**Comment by yuxu42** [comment]
@ceciliapeng2011 can we merge?

**Comment by xczhai** [comment]
@ceciliapeng2011 could you help merge this PR?

**Review by ceciliapeng2011 — APPROVED** [review]

**Comment by yuxu42** [comment]
@xczhai Could you please fix the CI issue and proceed the PR merege? Thanks!

**Comment by xczhai** [comment]
> @xczhai Could you please fix the CI issue and proceed the PR merege? Thanks!

Yes. fixed it.

**Pull request merged**